include/motorError.h

```diff
@@ -0,0 +1,15 @@
+#ifndef _MOTORERROR_H_
+#define _MOTORERROR_H_
+
+#include "telemetry.h"
+#include <string>
+
+enum ErrorType {None, Error1, Error3, Error4, Error5, Error6, Error8, Error9, Unknown};
```

Comment:
May be better to actually use the error types listed in the [spreadsheet](https://docs.google.com/spreadsheets/d/12O2UPdM_fqUVKd0IXZ638wb1obtBXQp11giv7swfWM4/edit#gid=1209182212). Don't need to use full names, but something more descriptive would be nice.